frontend/app/.server/web/exceptions/raoidc-session-invalid.exception.ts

```diff
@@ -0,0 +1,3 @@
+import { BaseWebException } from './base-web.exception';
+
+export class RaoidcSessionInvalidException extends BaseWebException {}
```

Comment:
Custom error classes in Javascript should override the base `Error.name` property, otherwise they will be misreported in stack traces.

So, add a constructor and inside that constructor do `this.name = 'RaoidcSessionInvalidException'`.

<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#custom_error_types>

Reply:
`BaseWebException` handle it.

```ts
export class BaseWebException extends Error {
  constructor(message: string, options?: ErrorOptions) {
    super(message, options);
    this.name = this.constructor.name; // Set the exception's name to the class name
    Error.captureStackTrace(this, this.constructor); // Captures the stack trace
  }
}
```

Reply:
https://stackoverflow.com/a/69901614

Reply:
I haven't seen this approach before. I'll have to trust that it works as you imply.